skill-recorder.lic

```diff
@@ -7,51 +7,68 @@ no_kill_all
 
```

Comment:
Hide whitespace to reduce the noise in the diff.

<img width="253" alt="image" src="https://github.com/user-attachments/assets/6c3d031c-8dc5-4a83-b029-c96d2bde42e1" />